src/components/video-page-category.js

```diff
@@ -142,15 +147,15 @@ function VideoListingPage(props) {
           <div className="like">
             <i
               class="fa fa-thumbs-up"
-              onClick={() => likedClickHandler(props.value)}
+              onClick={() => isLoggedIn ? likedClickHandler(props.value):navigate("/login")}
```

Comment:
suggestion: try to move onclick event to separate function and call the function here